src/gui_common/TweakedColourPicker.cs

```diff
@@ -0,0 +1,357 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Godot;
+
+/// <summary>
+///   Tweaked colour picker allows for tooltip texts translations and has a customized theming.
+/// </summary>
+public class TweakedColourPicker : ColorPicker
+{
+    /// <summary>
+    ///   This is where presets are stored after a colour picker exited scene tree.
+    /// <remarks>
+    ///   KeyValuePair&lt;GetPath(), GetPresets()&gt;:
+    ///   <ul>
+    ///     <li> First parameter (string) is the string format of NodePath; </li>
+    ///     <li> Second parameter (Color[]) is the preset colours to be stored. </li>
+    ///   </ul>
+    /// </remarks>
+    /// </summary>
+    private static readonly List<KeyValuePair<string, Color[]>> PresetsStorage
```

Comment:
Why is this a list and not a dictionary? The value is already a list of colours, so why not make this a dictionary as values with the same string could be combined to a single list of colours in a dictionary.